habitat/core/environments.py

```diff
@@ -78,3 +80,31 @@ def get_done(self, observations):
 
     def get_info(self, observations):
         return self.habitat_env.get_metrics()
+
+
+@habitat.registry.register_env(name="GymRegistryEnv")
```

Comment:
If the name of the key to register in registry coincides with the class name `name="GymRegistryEnv"` can be omitted.

I've searched the habitat-lab code and that is the common way (try searching `registry.register_`).